build-system/server/new-server/transforms/transform.ts

```diff
@@ -20,8 +20,9 @@ import posthtml from 'posthtml';
 import transformModules from './modules/modules-transform';
 import transformScriptPaths from './scripts/scripts-transform';
 import transformStories from './stories/stories-transform';
+import transformCss from './css/css-transform';
 
-const transforms = [transformStories, transformScriptPaths];
+const transforms = [transformStories, transformScriptPaths, transformCss];
```

Comment:
Curious: does the order of these transforms matter?

Reply:
yes it does. ill double check the internal transformers to mimic it closely